docs/content/troubleshooting/common-issues.md

```diff
@@ -477,4 +477,22 @@ Run your composer command beginning with...
 fin exec COMPOSER_MEMORY_LIMIT=-1 composer
 ```
 
-If you are still having memory issues, see [Out-of-memory Issues](/troubleshooting/common-issues/#issue-03).
\ No newline at end of file
+If you are still having memory issues, see [Out-of-memory Issues](/troubleshooting/common-issues/#issue-03).
+
+## Issue 19. Out of disk space
+
+Out of disk issues may manifest in many ways. One is:
+
+```
+ERROR 1114 (HY000) at line xxxx: The table 'xxxx' is full
+```
+
+You can diagnose this issue with `fin exec df -lh`. If a disk shows close to 100% usage it's time to expand it.
```

Comment:
Add a comma => If a disk shows close to 100% usage, it's time to expand it.

Reply:
```suggestion
You can diagnose this issue with `fin exec df -lh`. If a disk shows close to 100% usage, it's time to expand it.
```

Reply:
@shelane thanks! I've addressed this below.